apps/jira/jira-app/src/components/App/Steps/ContentTypeStep.tsx

```diff
@@ -61,9 +61,9 @@ const ContentTypeStep = ({
     );
   } else {
     contentToRender = (
-      <Flex>
+      <Flex flexDirection="column" gap="spacingM">
         {sortedContentTypes.map((ct) => (
-          <FormControl id={ct.name} key={ct.id}>
+          <FormControl id={ct.name} key={ct.id} style={{ marginBottom: 0 }}>
```

Comment:
We could use the marginBottom prop here to avoid adding inline styles with the style prop.  I think using the style prop would generally be a last ditch option.

```suggestion
          <FormControl id={ct.name} key={ct.id} marginBottom="none">
```